novuRajan/gannt-chart
# Edit add dynamic field

## Description of the problem this PR solves
Add fields need to be dynamic so that the user can add the required dynamic field

## Details of how the solution has been implemented
The addFormFields was made inside the config where we define label and all the other we need to add inside the form  

## Testing Instructions

```
 GanttChart.createGanttChart(data,{
        containerId: 'chart',
        viewMode: true,
        addFormFields: [
            {
                type: 'select',
                label: 'Type',
                name: 'type',
                options: [{
                        label:'Events',
                        value:'Event',
                        selected: true
                    },
                    {
                        label:'Vignette',
                        value:'Vignette'
                    }],
            },

        ],
    });
```

should be used where input type, name , id ,value and options can be added 

- click on add task button in the left side 
- you will find your added field in the form.

## Browsers tested
Select all browsers the solution was tested on <br>
*(test on as many as possible if the solution is front-end, otherwise this section is optional)*

## Notes
Any further information required for this PR to be reviewed and tested
